### PR TITLE
Update to FIDO2

### PR DESCRIPTION
**Pull request by yaraki**

### CODE REVIEW

**Comment by samtstern** [comment]
@agektmr I don't know this code I am only the owner in name, so I won't be reviewing unless you need it to merge.

**Comment by agektmr** [comment]
@samtstern sorry, I added you by mistake.

File: app/src/main/java/com/fido/example/fido2apiexample/SecurityTokenAdapter.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
+/*
+ *  Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (davidsun) [inline app/src/main/java/com/fido/example/fido2apiexample/SecurityTokenAdapter.java:2]: 2019 instead of 2017?

**Review by davidsun — APPROVED** [review]

File: app/src/main/java/com/fido/example/fido2apiexample/SecurityTokenAdapter.java (hunk 1 above)
Comment (yaraki) [reply app/src/main/java/com/fido/example/fido2apiexample/SecurityTokenAdapter.java:2]: Done. Thanks!

**Pull request merged**